hack/testing/test-upgrade.sh

```diff
@@ -196,7 +222,7 @@ addTriggers && \
 rebuildVersion "upgraded" && \
 
 upgrade "upgraded" && \
-verifyUpgrade true && \
+verifyUpgrade "upgraded" true && \
 
 ./e2e-test.sh $USE_CLUSTER && \
```

Comment:
1. don't need backslashes after `&&`
2. could get rid of all of these `&&` and the `|| return 1` mess above - the `&&` is what is making `-e` ineffective and we don't even need to do it. the point of `-e` is exactly what we're doing, where we want to quit if anything fails.